beacon-chain/core/blocks/block_operations.go

```diff
@@ -223,29 +241,6 @@ func ProcessRandao(
 	return beaconState, nil
 }
 
-// Verify that bls_verify(proposer.pubkey, hash_tree_root(get_current_epoch(state)),
-//   block.body.randao_reveal, domain=get_domain(state.fork, get_current_epoch(state), DOMAIN_RANDAO))
-func verifyBlockRandao(beaconState *pb.BeaconState, body *pb.BeaconBlockBody, proposerIdx uint64) error {
```

Comment:
Why is this deleted?

Reply:
Replaced with `verifySignature()`. Rather we have general functions than specific for each operation.